### PR TITLE
Adding a default CLI_UNDER_TEST environment variable under _common…

### PR DESCRIPTION
**Pull request by livarcocc**
Adding a default CLI_UNDER_TEST environment variable under _common.
Modifying runtests.sh and ps1 to put the variable at the head of PATH unless one is passed as a parameter.

This will run the tests against stage2 unless a parameter is passed to the test script.

### CODE REVIEW

**Comment by livarcocc** [comment]
@piotrpMSFT @brthor @davidfowl

**Comment by livarcocc** [comment]
@dotnet-bot Test Ubuntu Release Build please

File: scripts/test/runtests.sh [hunk 1]
```diff
@@ -81,4 +85,7 @@ done
 popd
 set -e
 
+#resetting the path so that we can be good neighbors to other scripts running after this.
+export PATH=$StartPath
```
Comment (brthor) [inline scripts/test/runtests.sh]: NO need for this in bash. Path will be local to this subshell and child shells only.

Comment (livarcocc) [inline scripts/test/runtests.sh]: Will it affect any scripts that run after this? Thats's what I was trying to avoid.

Comment (brthor) [inline scripts/test/runtests.sh]: Only scripts which are started by this script or source this script (which they shouldn't be) scripts which execute this script will be unaffected

Comment (livarcocc) [inline scripts/test/runtests.sh]: All right, I will remove it, though I will only add it back once your refactoring goes in.

**Comment by piotrpMSFT** [comment]
if this is still needed, rebase and reopen. @livarcocc

**Pull request closed without merging**